arch/cpu/cc26xx-cc13xx/Makefile.cc13xx

```diff
@@ -2,6 +2,6 @@ TI_XXWARE_PATH = lib/cc13xxware
 
 CONTIKI_CPU_SOURCEFILES += smartrf-settings.c prop-mode.c prop-mode-tx-power.c
 
-CFLAGS += -DCPU_FAMILY_CC13XX=1
```

Comment:
Hmm, is there a reason for the change (do we expect a new CC13 chip family)? I'm asking because just in the TSCH pull request I did use CPU_FAMILY_CC13XX in one place, this change means that it should be updated again.

Reply:
There is now CC13x0 and CC13x2 / CC26x0 and CC26x2

Reply:
I should add, happy to update this one here to reflect onto the TSCH PR which is kind of likely to get merged first?

Reply:
The TSCH PR uses the `CPU_FAMILY_CC13XX` flag to determine whether the radio is running with the the slow bitrate (currently assumed to be 50 kbps, which is of course not the most flexible solution).  

Help me to understand - does this Makefile.cc13xx refer only to CC13x0 chips? Also, looks like CC13x2 still has the proprietary Sub-GHz radio.

So, perhaps leave the `-DCPU_FAMILY_CC13XX=1` line as is, and also add `-DCPU_FAMILY_CC13X0=1` for the CC13x0 in particular?

Reply:
It is confusing yes. This CPU dir supports what is now called CC26x0 and CC13x0.

The `simplelink-cc13xx-cc26xx` dir supports all chips of the family, including the various new CC26x2 and CC13x2. The new ones being CM4s.

Happy to undo the diff or to apply your proposal.